packages/build/src/steps/run_step.ts

```diff
@@ -155,7 +154,7 @@ export const runStep = async function ({
     systemLog,
     quiet,
   })
-  return { ...newValues, newIndex: index + 1 }
+  return { ...newValues }
```

Comment:
Not sure we need to create a new object from `newValues`? You can return `newValues` directly.